src/components/Board.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { useContext } from "react";
+import { GameContext } from "../context/GameContext";
+import { startGame } from "../logic/actions";
+
+const Board = (): React.ReactElement => {
+  const { state, dispatch } = useContext(GameContext);
+
+  return (
+    <>
+      <div>{JSON.stringify(state)}</div>
+      <button onClick={(): void => dispatch(startGame())}>Start</button>
```

Comment:
I don't think you need `: void` here right? It should be inferred, and it doesn't look like its used, but then idk how jsx handles types.

Reply:
This was eslint getting worked out about it >.<

Reply:
Hmm if eslint says returned values are required, then such is required. (for better or worse)